### ISSUE TITLE
A stop job is running for User Manager for UID 1000 - Portmaster slows down shutdown/reboot process

### ISSUE THREAD

**Issue by Hell01World** [safing/portmaster#822]
**What happened**:

In the shutting down and rebooting process, systemd will output "A stop job is running for User Manager for UID 1000" and after around ~20 seconds it's going to stop it. 

**What did you expect to happen?**:

For Portmaster to not slow down shudown/reboot process

**How did you reproduce it?**:
1. Shutdown/Reboot
2. Systemd outputs "A stop job is running for User Manager for UID 1000"
3. After ~20 seconds it's going to stop it

**Workaround**:

Workaround for this would be this command: 'nmcli radio wifi off && sudo systemctl stop portmaster ; systemctl reboot'

**Install Information**:

Distribution: Arch Linux
Kernel: 5.19.5-arch1-1 x86_64
Display server: Xorg
Desktop enviroment: Xfce
Display manager: none (running exec startxfce4 from .xinitrc at boot)
Using NetworkManager for managing network
Installed Portmaster from the [PKGBUILD](https://docs.safing.io/portmaster/install/linux#arch-linux)

**Debug Information**:

[Portmaster log](https://pastebin.com/EZJb5gqz)

**Comment by NubsParrot** [comment]
Try this https://codybonney.com/change-default-timeouts-starting-stopping-systemd-units/. This could happen with any service/program might not be tied with Portmaster. be sure to star my projects thank u.

**Comment by dhaavi** [comment]
Hey @Hell01World, thanks for the report.

This can usually happen when there is an internal issue with Portmaster. One of its internal modules might not be able to shut down for some reason. If you encounter this again, quickly open your browser and go to http://127.0.0.1:817/api/v1/debug/core?style=github -  this will return the debug info and help us see where Portmaster is hanging.

I'll mark this issue as "waiting for input". It will close after a month, but just reopen and submit the data when it happens again.

**Comment by Hell01World** [comment]
Hi @dhaavi, 
a few days ago I reinstalled Arch and once I installed Portmaster and enabled it's service, the issue seems to... solved it's self? Perhaps some kind of bug in one of the packages that Portmaster used was fixed in the recent update, or I misconfigured something (which I doubt). I'm going to close this issue for now and reopen it once the issue will come back, if ever. Anyway, thank you for your time and keep up the amazing project :)
